lib/midi.coffee

```diff
@@ -8,19 +8,36 @@ L = utils = midi = midi_in = ___ = null
   midi = rtMidi
   ___ = utils.____ '[midi]'
 
+# Parse method copied from https://github.com/hhromic/midi-utils-js/blob/master/midiparser.js#L418
```

Comment:
That is a really good point. What is the appropriate way of indicating that this is modified from hhromic's code? I'm not sure that just including the [MIT license information from his project](https://github.com/hhromic/midi-utils-js/blob/master/LICENSE) in this file is appropriate since only parts of this function come from his project.